common/script/content/appearance/backgrounds.js

```diff
@@ -320,4 +320,4 @@ forOwn(backgrounds, function prefillBackgroundSet (value) {
   });
 });
 
-export default backgrounds;
+module.exports = backgrounds;
```

Comment:
Babel changed how it treats exports from v5 to v6.

```js
// babel 5

// default is equivalent with module.exports
export default backgrounds;
module.exports = backgrounds;

import backgrounds from './backgrounds';
let backgrounds = require('./backgrounds');

// also equivalent
export foo;
export bar;
module.exports.foo = foo;
module.exports.bar = foo;

import { foo, bar } from './foobar';
let { foo, bar } = require('./foobar');

// also equivalent
export default {
  foo,
  bar,
}
module.exports = {
  foo,
  bar,
}

import { foo, bar } from './foobar';
let { foo, bar } = require('./foobar');
```


```js
// babel 6

// default is not equivalent with module.exports
export default backgrounds;
module.exports = {
  default: backgrounds,
};

import backgrounds from './backgrounds';
let backgrounds = require('./backgrounds').default;

// these are the same as before, I believe
export foo;
export bar;
module.exports.foo = foo;
module.exports.bar = foo;

import { foo, bar } from './foobar';
let { foo, bar } = require('./foobar');

// this is not the same
export default {
  foo,
  bar,
}
module.exports = {
  default: {
    foo,
    bar,
  },
}

import { foo, bar } from './foobar';
let { foo, bar } = require('./foobar').default;
```

I chose to go with `module.exports` instead of `export default`, because it was easier for me to wrap my head around (and required fewer changes to files). It's a little weird to mix import syntax with module.exports, but as long as we're consistent, I think it's okay.